cpp/src/cylon/join/sort_join.cpp

```diff
@@ -172,20 +154,10 @@ static inline arrow::Status do_sorted_join(const std::shared_ptr<arrow::Table> &
   // clear the sort columns
   left_index_sorted_column.reset();
   right_index_sorted_column.reset();
-  t2 = std::chrono::high_resolution_clock::now();
-  LOG(INFO) << "Index join time : "
-            << std::chrono::duration_cast<std::chrono::milliseconds>(t2 - t1).count();
-  LOG(INFO) << "Building final table with number of tuples - " << left_indices.size();
-
-  t1 = std::chrono::high_resolution_clock::now();
```

Comment:
shall we wrap these around a `ifdef CYLON_DEBUG`?

Reply:
I think it will make the code too messy. I think we don't need these unless we are writing a research paper. In that case, someone can add these and measure.